tests/src/strategy.spec.ts

```diff
@@ -34,6 +79,7 @@ describe('Strategy with passport', () => {
     // It only provides a meaning when served with a 3xx (redirection) or 201 (created) status response.
     expect(response.statusCode).toBe(302);
     const redirectURL = new URL(response.header.location);
+    console.log(response.headers.location);
```

Comment:
extra console.log

Reply:
POC 🙂 will remove once this methods is feasible for all